resources/qml/MessageInput.qml

```diff
@@ -133,6 +136,7 @@ Rectangle {
                 padding: 0
                 topPadding: 8
                 bottomPadding: 8
+                leftPadding: inputBar.showAllButtons? 0 : 8
```

Comment:
Shouldn't you be modifying the margins of the other button instead?

Reply:
It shouldn't actually matter. i felt it made the most sense to have the padding in the element that's visible, but it's also applied if visible: false.

Reply:
Mhm, I guess, but now we mix paddings and margins and the layout is even more of a mess. Not really your fault, but it also doesn't get better :D